docs/virtual_machines/interfaces_and_networks.md

```diff
@@ -518,6 +518,56 @@ fields.
 More information about SLIRP mode can be found in [QEMU
 Wiki](https://wiki.qemu.org/Documentation/Networking#User_Networking_.28SLIRP.29).
 
+#### Slirp network binding plugin
+Since v1.1.0, Kubevirt delegate Slirp network configuration to a hook sidecar using Slirp network binding plugin by default.
+
+In case no image is registered, Kubevirt will use the following default image: `quay.io/kubevirt/network-slirp-binding:20230830_638c60fc8`.
+
+> **Note:** In the next release (v1.2.0) no default image will be set by Kubevirt, registering an image will be mandatory.
+
+> **Note:** On disconnected clusters it will be necessary to mirror Slirp binding plugin image to the cluster registry.
+
+### Network binding plugins
+Since v1.1.0, Kubevirt enables delegating network binding configurations to a hook sidecar.
+For more information about Kubevirt hook sidecar please refer to https://github.com/kubevirt/kubevirt/tree/v1.0.0/cmd/example-hook-sidecar
```

Comment:
But we are not using the user facing API described there, it may confuser a common reader. We are not even requiring the FG.

Maybe it is better to ref the slirt sidecar instead.

Reply:
I removed the reference for the sidecar example for now, once we have proper doc for hook sidecars I will add reference for it.